### PR TITLE
init repository

### PR DESCRIPTION
**Pull request by tudm-0884**
https://edu-redmine.sun-asterisk.vn/issues/11822

### CODE REVIEW

**Comment by tudm-0884** [comment]
ready

File: app/Repositories/Contracts/TrainerRepository.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+{
+	/**
+     * Create a new Repository instance.
+     *
+     * @param  TrainerRepositoryInterface
+     * @return void
+     */
+	public function __construct(Trainer $model)
+	{
+		parent::__construct($model);
```
Comment (BunnyPi04) [inline app/Repositories/Contracts/TrainerRepository.php]: indent = 4 spaces

**Comment by unamed1111** [comment]
ready

**Comment by tudm-0884** [comment]
ready

**Pull request merged**